### PR TITLE
NSPR-8172 monitor kernel clock shutdown status on U2 card

### PR DESCRIPTION
**Pull request by rajkumar-xilinx**
Add support in xrt to monitor the clock shutdown status in U2 card

Signed-off-by: Rajkumar Rampelli <rajkumar@xilinx.com>

### CODE REVIEW

File: src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-core.c [hunk 1]
```diff
@@ -493,6 +493,8 @@ static int health_check_cb(void *data)
 
 	(void) xocl_clock_status(lro, &latched);
 
+	xocl_xmc_clock_status(lro, &latched);
+
```
Comment (chvamshi-xilinx) [inline src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-core.c:497]: Can you please add a comment explaining why do we need to add a separate logic for U.2. example: U.2 doesn't use UCS etc

**Review by chvamshi-xilinx — APPROVED** [review]
Logic looks good to me.. Please add some comments explaining the function.

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c [hunk 2]
```diff
@@ -3877,6 +3880,26 @@ static int xmc_access(struct platform_device *pdev, enum xocl_xmc_flags flags)
 		raptor_cmc_access(pdev, flags);
 }
 
+static void clock_status_check(struct platform_device *pdev, bool *latched)
```
Comment (chvamshi-xilinx) [inline src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c:3883]: Same here. Please add some comments about this function. As CMC for U.2 is different, its good to add description about U.2 CMC behavior.

**Comment by dayeh-xilinx** [comment]
retest this please

**Pull request merged**